website/addons/wiki/templates/log_templates.mako

```diff
@@ -17,3 +17,27 @@ deleted wiki page
 <a data-bind="attr: {href: wikiUrl}, text: params.page"></a>
 of <a class = "log-node-title-link overflow" data-bind="text:nodeTitle, attr: {href: nodeUrl}"></a>
 </script>
+
+<script type="text/html" id="made_wiki_public">
+made the wiki
+of <a class = "log-node-title-link overflow" data-bind="text:nodeTitle, attr: {href: nodeUrl}"></a>
+publicly editable
+</script>
+
+<script type="text/html" id="made_wiki_public_no_user">
```

Comment:
This format is used multiple different places including in the logs for making projects public/private. (website/templates/log_templates.mako)